app/src/main/java/com/keylesspalace/tusky/components/login/LoginActivity.kt

```diff
@@ -320,10 +314,13 @@ class LoginActivity : BaseActivity(), Injectable {
             )
 
             val intent = Intent(this, MainActivity::class.java)
-            intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
```

Comment:
as per documentation: 
> Also note that this method can only be used to customize cross-activity transitions but not cross-task transitions which are fully non-customizable as of Android 11.

Since we finish the starting activity, removing these flags shouldn't be a problem

Reply:
can we get to login activity for non-top-level activity?

Reply:
ah but we `finishAffinitiy()`  now, we are good